### PR TITLE
Export 2019 03 12

### PR DESCRIPTION
**Pull request by benjamin-cheng**

### CODE REVIEW

File: locales/templates/app.pot [hunk 1]
```diff
@@ -417,11 +447,11 @@ msgid "Use %1$s button"
 msgstr ""
 
 msgctxt "settings_list_sorting"
-msgid "Prompt to sort"
+msgid "Ask to sort"
 msgstr ""
 
 msgctxt "settings_detail_sorting"
-msgid "Always prompt to add screenshot to a collection"
+msgid "After you take a screenshot, always ask you which collection to put it in."
```
Comment (Delphine) [inline locales/templates/app.pot]: I think this sentence should be rephrased. 
I'm thinking you want to drop the "you" in "always ask you which collection(...)"? Or something else?

**Review by Delphine — CHANGES_REQUESTED** [review]

File: locales/templates/app.pot (hunk 1 above)
Comment (ChenMorpheus) [reply locales/templates/app.pot]: The copy was provided by a native speaker. But I think "After you take a screenshot, always ask which collection to put it in." is better. Thanks.

**Comment by benjamin-cheng** [comment]
Hi @Delphine, we've updated the files to address your comments, could you please help to review? Thank you!

**Review by Delphine — APPROVED** [review]
LGTM!
thanks :)

**Pull request merged**